test/podman-tests.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+# Do not set -e as we want to work on all results of `rpm -q dnf[5]`.
+set -exo pipefail
+
+if [ "$(id -u)" -ne 0 ];then
+    echo "Please run this script as superuser"
+    exit 1
+fi
+
+if [[ $1 == '' ]]; then
```

Comment:
The conventional way to test this is `if [[ -z "$1" ]].....`

Also, please define named variables for these: 
```bash
TEST_TYPE=$1
STREAM=$2
```
Then all the code below can work using those names instead of the unclear `$1/$2`